util/util.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+)
+
+// SplitCmd split full command to namespace, command, version.
+// ex(ns/cmd/1.0.1 => ns cmd 1.0.1)
+func SplitCmd(cmd string) (namespace, command, version string, err error) {
+	splitNamespce := strings.Split(cmd, "/")
```

Comment:
How about using regular expression to split and validate?

Reply:
That would be better. I will use regular expression.